models/sessions/scratch/snowplow_mobile_sessions_aggs.sql

```diff
@@ -13,7 +13,7 @@
 with events as (
   select
     es.session_id,
-    es.event_id,
+    cast(es.event_id as {{ dbt_utils.type_string() }}) as event_id,
```

Comment:
I hadn't appreciated that there was `uuid` data type in Postgres before. It seems like this is preferable to strings in terms of performance and storage. Sorry to flip flop but I think we should keep it as `uuid` if possible. Is it just the `session_last_event_id` calc below that is causing issues with the uuid data type? If so, perhaps we have it as you had it before but recast it to uuid once calculated?

Reply:
I think it's a less elegant solution but I appreciate the value of keeping the `uuid` data type, so I've changed it back although I'm utilising the `dbt_utils.type_string()` which I wasn't doing before these changes!